apps/communications/contacts/js/activities.js

```diff
@@ -216,9 +217,46 @@ var ActivityHandler = {
     }
   },
 
+  // Generates a name for the contact returned as a vcard
+  _getContactName: function(theContact) {
```

Comment:
I think this method has more sense into the contact2vcard file, as an utility method, not to be used by the rest of activities.